htdocs/core/class/html.form.class.php

```diff
@@ -6957,7 +6957,7 @@ public function load_tva($htmlname = 'tauxtva', $selectedrate = '', $societe_ven
 
 		if ($num > 0) {
 			// Definition du taux a pre-selectionner (si defaulttx non force et donc vaut -1 ou '')
-			if ($defaulttx < 0 || dol_strlen($defaulttx) == 0) {
```

Comment:
Change is suspicious:
If we got a vat rate that is '' ur null, it leans we failed to find a vat rate in database, so we can fallback on the option.
But if we got 0, it leans we found a value that is '0', so old test seems good.
Can you describe how to reproduce the bug ?